services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/structure/file/FileOnDisk.java

```diff
@@ -61,5 +61,8 @@ public void setBytes(byte[] bytes) {
    */
   @Override
   public void prepare(ImmutableStack<Object> indices) {
+    /*
+    Method override exists here to comply with the implementation rules for the Writable interface.
```

Comment:
``// Method override exists here to comply with the implementation rules for the Writable interface.``